src/puppet.js

```diff
@@ -653,6 +780,11 @@
     }
 
     this.queue.receive(this.obj, patches);
+    if(this.queue.pending && this.queue.pending.length && this.queue.pending.length > 1) {
```

Comment:
This number should not be hard-coded here.

Plus, I do not think `1` is the good number. I'm afraid it may interrupt our keystroke integration.